docs.md

```diff
@@ -55,6 +55,23 @@ Each specification consists of:
 * **on**: One or more of these. Each on is an action. The act. More than one on indicates that they share the same context.
 * **it**: One or more of these. Each it is a consequence of the action. The assert. Normally it's good practice to have a single assertion per it.
 
+#### Dependency
+
+Adding the dependency to Spek is needed to use Spek.
+
+For example, gradle users add the following lines to your `build.gradle`
+
+```groovy
+repositories {
+  maven {
+    url "http://repository.jetbrains.com/all"
+  }
+}
+dependencies {
+  testCompile "org.jetbrains.spek:spek:$your_spek_version" //replace $your_spek_version with correct Spek version (e.g. 0.1.195)
```

Comment:
Well, repo doesn't have tags so the only way to check if you're on latest
version is to open maven repo and find it out which is not very convenient.

On Wed, 1 Jun 2016, 09:29 Hadi Hariri, <notifications@github.com> wrote:

> In docs.md
> <https://github.com/JetBrains/spek/pull/73#discussion_r65308031>:
>
> > @@ -55,6 +55,23 @@ Each specification consists of:
> >  * **on**: One or more of these. Each on is an action. The act. More than one on indicates that they share the same context.
> >  * **it**: One or more of these. Each it is a consequence of the action. The assert. Normally it's good practice to have a single assertion per it.
> >
> > +#### Dependency
> > +
> > +Adding the dependency to Spek is needed to use Spek.
> > +
> > +For example, gradle users add the following lines to your `build.gradle`
> > +
> > +```groovy
> > +repositories {
> > +  maven {
> > +    url "http://repository.jetbrains.com/all
>
> "
> > +  }
> > +}
> > +dependencies {
> > +  testCompile "org.jetbrains.spek:spek:$your_spek_version" //replace $your_spek_version with correct Spek version (e.g. 0.1.195)
>
> Yeah but it changes every so often so does it really make sense?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/JetBrains/spek/pull/73/files/2ecbba9a57981a19222667beefbd7392c81e8b58#r65308031>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA7B3IDnWk4tk3RBjvz21UYRt_WQdbz_ks5qHSbcgaJpZM4H66PP>
> .
>